redbot/cogs/admin/announcer.py

```diff
@@ -5,9 +5,8 @@
 
 
 class Announcer:
-    def __init__(self, ctx: commands.Context,
-                 message: str,
-                 config=None):
+
+    def __init__(self, ctx: commands.Context, message: str, config=None):
```

Comment:
Here it does not add spaces around the `=` operator, despite doing it above frequently

Reply:
One of them has a type annotation and the other does not. It's consistent and compliant.